projects/step-frontend/src/lib/modules/execution/components/schedule-overview/schedule-overview.component.ts

```diff
@@ -0,0 +1,36 @@
+import { Component, inject, OnInit } from '@angular/core';
+import { ActivatedRoute } from '@angular/router';
+import { AugmentedSchedulerService, ExecutiontTaskParameters } from '@exense/step-core';
+
+@Component({
+  selector: 'step-schedule-overview',
+  templateUrl: './schedule-overview.component.html',
+  styleUrls: ['./schedule-overview.component.scss'],
+})
+export class ScheduleOverviewComponent implements OnInit {
+  private _api = inject(AugmentedSchedulerService);
```

Comment:
As we will later also have other API services in this controller (at least AugmentedExecutionSerivice) we should already start naming the services with good names like _scheduleApi